### PR TITLE
create new api for login from mobile app

### PR DESCRIPTION
**Pull request by fuentesloic**
I tried ... even not sure if I understand my own python ^^

### CODE REVIEW

File: api/views.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import json
 
 from flask import request, Response
```
Comment (vinyll) [inline api/views.py]: `from flask import request, Response, jsonify`

File: api/views.py [hunk 2]
```diff
@@ -37,3 +37,12 @@ def api_user(id):
...
     return Response(json.dumps(data), mimetype='application/json')
+
+
+@app.route('/api/login/', methods=['get', 'post'])
+def api_login():
+    response = request.json
+    user = User.objects.get(email=request.form['email'])
+    if user.check_password(password=request.form['password']):
+        login_user(user)
+        return Response(json.dumps(user), mimetype='application/json')
```
Comment (vinyll) [inline api/views.py]:
```python
    return jsonify(user)
else:
    return jsonify({'success': False})
```

File: api/views.py [hunk 3]
```diff
@@ -37,3 +37,12 @@ def api_user(id):
     if 'email' in data['socials']:
         data['socials']['email'] = 'hidden'
     return Response(json.dumps(data), mimetype='application/json')
+
+
+@app.route('/api/login/', methods=['get', 'post'])
+def api_login():
+    response = request.json
```
Comment (vinyll) [inline api/views.py]: seems like you're not using this

**Review by vinyll — CHANGES_REQUESTED** [review]
Good job mate!

**Comment by fuentesloic** [comment]
I'm more like a child using copy/paste than python dev 😛

**Comment by fuentesloic** [comment]
@vinyll thx to have fixed python!!!

**Review by vinyll — DISMISSED** [review]

**Review by vinyll — APPROVED** [review]
@fuentesloic Notice that when an error occurs, a status code of 400 or 401 is returned.
When status code is 200, it means it's successful and the response contains the JSON user object :)

**Comment by fuentesloic** [comment]
got it!

**Pull request merged**